microshift_install/microshift-greenboot.adoc

```diff
@@ -6,13 +6,13 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Greenboot is the generic health check framework for the `systemd` service on RPM-OSTree-based systems. The `microshift-greenboot` RPM and `greenboot-default-health-check` are optional RPM packages you can install. Greenboot is used to assess system health and automate a rollback to the last healthy state in the event of software trouble.
+Greenboot is the generic health check framework for the `systemd` service on RPM-OSTree-based systems. The `microshift-greenboot` RPM and `greenboot-default-health-check` are RPM packages installed with {product-title}. Greenboot is used to assess system health and automate a rollback to the last healthy state in the event of software trouble.
```

Comment:
Forgive me if this is a dumb question, should these mentions of `greenboot-default-health-check` be `greenboot-default-health-checks` based on https://github.com/openshift/openshift-docs/pull/63057/?

Reply:
@mburke5678 Yes, but I am not correcting it here also or it may cause the cherry pick to fail.